overlord/snapstate/conflict.go

```diff
@@ -26,6 +26,9 @@ import (
 	"github.com/snapcore/snapd/overlord/state"
 )
 
+// FinalTasks are task kinds for final tasks in a change which means no further change work should be performed afterward, usually these are tasks that commit a full system transition.
+var FinalTasks = map[string]bool{"mark-seeded": true, "set-model": true}
```

Comment:
I understand the map is more convenient but []string seems more natural for a global like this.